### PR TITLE
Optimize get_schema code

### PR DESCRIPTION
**Pull request by spolu**

### CODE REVIEW

**Comment by spolu** [comment]
@fontanierh happy to push this forward as discussed or leave to you. Let me know :+1:

**Comment by spolu** [comment]
I think we want to remove table_id from DatabaseRow and make the row_id optional so that we can use DatabaseRow everywhere we work with "rows"

**Comment by spolu** [comment]
Pushed a commit in that direction

**Comment by fontanierh** [comment]
> @fontanierh happy to push this forward as discussed or leave to you. Let me know 👍

Feel free to merge, this looks a lot better. I'll take care of rebasing & adapting the query PR to follow the same guidelines

File: core/src/databases/database.rs [hunk 1]
```diff
@@ -239,9 +193,9 @@ impl DatabaseSchemaTable {
         DatabaseSchemaTable { table, schema }
     }
 
-    pub fn table(&self) -> &DatabaseTable {
-        &self.table
-    }
+    // pub fn table(&self) -> &DatabaseTable {
```
Comment (fontanierh) [inline core/src/databases/database.rs:196]: 🧹

**Review by fontanierh — DISMISSED** [review]

**Pull request merged**